CMakeLists.txt

```diff
@@ -589,8 +590,12 @@ if (ISPC_PREPARE_PACKAGE)
 
     # CPack configuration
     if (WIN32)
-        set(CPACK_GENERATOR "ZIP")
+        set(CPACK_GENERATOR "WIX")
         set(ISPC_SYSTEM_NAME "windows")
+        # WIX generator expects version in format x.y.z where x, y, z are numbers
+        set(CPACK_PACKAGE_VERSION "${ISPC_VERSION_MAJOR}.${ISPC_VERSION_MINOR}.${ISPC_VERSION_PATCH}")
+		set(CPACK_WIX_UPGRADE_GUID "EDD858F2-19B0-4E5B-B9B9-2A09B85D451C")
+		set(CPACK_WIX_CMAKE_PACKAGE_REGISTRY "Intel(R) Implicit SPMD Program Compiler")
```

Comment:
I think the package name should be `ISPC`, so that `find_package(ISPC)` would return this package.

Though, I might be reading docs incorrectly. Could you please check if this works?

Reply:
Since we do not provide cmake configuration file as part of our installation, find_package(ISPC) will not work.